fuse/macfuse/Portfile

```diff
@@ -61,23 +61,40 @@ build {
     }
 }
 
-set dir /Library/Filesystems/macfuse.fs
+set fs_dir /Library/Filesystems
+set dir ${fs_dir}/macfuse.fs
 
 destroot {
     copy ${pkg}/Library/Filesystems ${destroot}${prefix}/Library
     xinstall -W ${pkg}${dir}/Contents/Resources -m 4755 load_macfuse mount_macfuse \
         ${destroot}${prefix}${dir}/Contents/Resources
+    delete {*}[glob ${destroot}${prefix}${dir}/Contents/Resources/*.app]
+    delete ${destroot}${prefix}${dir}/Contents/_CodeSignature
```

Comment:
I believe I didn't delete these files to not invalidate the signature. IMO remove this for now and just keep the `fs_link` change in this PR.

Reply:
I believe only the signature of kexts is validated, deleting the signature in the fs bundle shouldn't matter. I'll spin up a vm and test real quick

Reply:
TBH I'd rather not mess with signatures, especially since this is outside the MacPorts root.